compaction/compaction.cc

```diff
@@ -1707,15 +1755,18 @@ static future<compaction_result> scrub_sstables_validate_mode(sstables::compacti
 
     clogger.info("Finished scrubbing in validate mode {} - sstable(s) are {}", sstables_list_msg, valid ? "valid" : "invalid");
 
+    uint64_t validation_errors = 0;
     if (!valid) {
         for (auto& sst : *sstables->all()) {
             co_await sst->move_to_quarantine();
+            ++validation_errors;
```

Comment:
Why do we update this counter here?

Reply:
In validate mode we do not enter any of on_invalid_* functions, so the errors should be counted somewhere else. This seems to be reasonable.

Reply:
But this will be increase the counter once for each sstable, whereas the validation can find many more in each sstable.